v2/emailpassword/custom-ui/init/user-management-dashboard/setup.mdx

```diff
@@ -257,15 +257,19 @@ init(
 
 ## Content Security Policy
 
-If you return a `Content-Security-Policy` header in from your backend, you will need to include the following directives for the user management dashboard to work correctly
+If your backend returns a `Content-Security-Policy` header, you will encounter the following UI displaying the CSP violation details. Follow the instructions provided in this UI to make necessary adjustments to your backend CSP configuration.
 
+![CSP error handled UI](/img/dashboard/csp-error.png)
```

Comment:
for dashboard for python / golang, this is not true right?

Reply:
yeah you are right

Reply:
Fixed it by added SDK tabs UI to show SDK specific docs.